### PR TITLE
Fix GridPro setEnabled

### PR DESCRIPTION
**Pull request by shadikhani**
Fix #25

### CODE REVIEW

File: src/vaadin-grid-pro-inline-editing-mixin.html [hunk 1]
```diff
@@ -43,6 +43,10 @@
         singleCellEdit: {
           type: Boolean,
           notify: true // FIXME(yuriy-fix): needed by Flow counterpart
+        },
+
+        _disableEditing: {
```
Comment (tomivirkki) [inline src/vaadin-grid-pro-inline-editing-mixin.html]: "disableEditing" sounds like a function whereas this is a property. The naming should be aligned between properties; similarly to "disabled", this should be "_editingDisabled"

File: test/edit-column.html [hunk 2]
```diff
@@ -544,6 +544,25 @@
         });
       });
 
+      describe('disable editing', () => {
+
+        let grid, input;
```
Comment (tomivirkki) [inline test/edit-column.html]: `input` is declared in the "describe" block but it's not assigned in "beforeEach". Either assign it in "beforeEach" or remove it and use `const input = ...` in the test case.

File: test/edit-column.html [hunk 3]
```diff
@@ -544,6 +544,25 @@
...
+        beforeEach(() => {
+          grid = fixture('default');
+          grid.items = getItems();
+          flushGrid(grid);
+        });
+
+        it('should not show editor when editing is disabled', () => {
+          const cell = getContainerCell(grid.$.items, 1, 0);
+          grid._disableEditing = true;
+          enter(cell);
```
Comment (tomivirkki) [inline test/edit-column.html]: Only the "keydown" case is tested even though changes were made to "click" and "dblclick" also. Either add tests for all the cases or do the `if (this._disableEditing) { return }` check in one central place (`_enterEditFromEvent` maybe?)

File: src/vaadin-grid-pro-inline-editing-mixin.html (hunk 1 above)
Comment (shadikhani) [reply src/vaadin-grid-pro-inline-editing-mixin.html]: Done.

File: test/edit-column.html (hunk 3 above)
Comment (shadikhani) [reply test/edit-column.html]: Done.

**Pull request merged**